src/plugins/kibana_usage_collection/server/collectors/usage_counters/rollups/rollups.ts

```diff
@@ -62,7 +63,7 @@ export async function rollUsageCountersIndices(
     );
 
     return await Promise.all(
-      docsToDelete.map(({ id }) => savedObjectsClient.delete(USAGE_COUNTERS_SAVED_OBJECT_TYPE, id))
+      docsToDelete.map(({ id, type }) => savedObjectsClient.delete(type, id))
```

Comment:
Q: do we need to specify the namespace when deleting it

Reply:
Q2: Should we take this opportunity to use bulkDelete?

Reply:
Will update to take namespaces into account.
Unfortunately `bulkDelete` does not allow for a multi-namespace delete operation, and I believe it relies on the spaces extension.